packages/bitcoin/src/fees/bitcoin-fees.ts

```diff
@@ -2,13 +2,12 @@ import { AverageBitcoinFeeRates, Money } from '@leather.io/models';
 
 import {
   CoinSelectionRecipient,
-  CoinSelectionUtxo,
   DetermineUtxosForSpendArgs,
   determineUtxosForSpend,
   determineUtxosForSpendAll,
 } from '../coin-selection/coin-selection';
 
-type GetBitcoinTransactionFeeArgs = DetermineUtxosForSpendArgs & {
+type GetBitcoinTransactionFeeArgs = DetermineUtxosForSpendArgs<{ value: string; txid: string }> & {
```

Comment:
Should we just add a type for `{ value: string; txid: string }`?

Reply:
Don't worry, it's not a big deal. It's actually probably easier to read if repeated than if it was abstracted